### ISSUE TITLE
No icon in taskbar with v1.8 on Windows 7

### ISSUE THREAD

**Issue by TerDale** [xmichelo/Beeftext#132]
### Expected behavior
The application icon should be displayed in Windows 7 taskbar upon app launch

### Actual behavior
A generic icon is displayed instead (see attachment)

### Steps to reproduce the behavior
Just launch the app

### Operating system
Windows 7 SP1

![image](https://user-images.githubusercontent.com/1826224/53942526-49a31f80-40bb-11e9-831f-447b9310d5fc.png)

**Comment by TerDale** [comment]
I precise that I checked with the provided button in "Preferences" that I have the latest version.

**Comment by xmichelo** [comment]
Hi @TerDale 

It's strange. I have a VM of Windows 7 (Ultimate version, SP1 build 7601), I set the taskbar to have small icons, and the icon is correctly displayed. 

![sevensmallicon](https://user-images.githubusercontent.com/1926756/54068957-763d6f80-4252-11e9-9f75-9a440c85ceee.png)

I cared to provide every icon size recommended by Microsoft, and this being a minor annoyance on a system that will reach EOL in less than a year, you'll understand I will not spend time on the issue, so I'm closing it.
